applications/luci-app-softether/luasrc/view/softether.htm

```diff
@@ -0,0 +1,74 @@
+<%+header%>
+<h2>SoftEther Status</h2> 
+<% 
+  local function table_merge(table1, table2)
+    for k,v in pairs(table2) do
+      table1[k] = v
+    end
+    return table1
+  end
+
+  local function get_account_list()
+    local accounts = {}
+    local ac_list = io.popen("vpncmd localhost /client /csv /cmd:accountlist | tail -n +4 2> /dev/null ")
+    if ac_list then
+      local line
+      for line in ac_list:lines() do
+        local line = string.split(line, ",")
+        local d = {}
+        d["Status"] = line[2]
+        accounts[line[1]] = d
+      end
+    end
+    return accounts
+  end
+
+  local function get_account_status(ac_name)
+    local t = {}
+    local csv = io.popen("vpncmd localhost /client /csv /cmd:accountstatusget " .. ac_name ..  " | tail -n +4 2> /dev/null ")
+    if csv then
+      local line
+      for line in csv:lines() do
+        local line = string.split(line, ",")
+        if table.getn(line) == 2 then
+          t[line[1]] = line[2]
+        end
+      end
+    end
+    return t
+  end
+
+
+  local function dump_account(account, properties)
+   luci.http.write("<div class=\"tr cbi-section-table-row\">")
+   luci.http.write("<div class=\"td\" style=\"width: 20%;vertical-align: top;\"><strong>Account:</strong> " .. account .. "</div>")
+   luci.http.write("<div class=\"td\">")
+   for i,v in pairs(properties) do
+     luci.http.write("<strong>" .. i .. ":</strong> " .. v .. "</br>")
+   end
+   luci.http.write("</div>") -- td
+   luci.http.write("</div>") -- tr
+ end
+
+
+-- Gather stuff
+ local accounts = get_account_list()
+ for acc_name, acc_properties in pairs(accounts) do
+   local acc_status = get_account_status(acc_name)
+   table_merge(acc_properties, acc_status)
+ end
+
+-- Render stuff
+  luci.http.write("<div class=\"cbi-section\">")
+  luci.http.write("<div class=\"cbi-section-node\">")
+  luci.http.write("<div class=\"table cbi-section-table\">")
+
+  for acc_name, acc_properties in pairs(accounts) do
+    dump_account(acc_name, acc_properties)
+  end
+  luci.http.write("</div>") -- table
+  luci.http.write("</div>") -- section-node
+  luci.http.write("</div>") -- section
+
+ %>
+<%+footer%>
```

Comment:
This file should be converted to tab.
Now use spaces at the beginning of the lines.
Tabs saves memory here if it is not minified

Reply:
Indentation is now donw with tabs.